packages/nc-gui/components/smartsheet/Grid.vue

```diff
@@ -566,7 +566,7 @@ watch(
                       <span class="flex-1" />
 
                       <div
-                        v-if="(!readOnly || hasRole('commenter', true) || hasRole('viewer', true)) && !isLocked"
+                        v-if="(!readOnly || hasRole('commenter', true) || hasRole('viewer', true))"
```

Comment:
parentheses are not necessary. 

```
v-if="!readOnly || hasRole('commenter', true) || hasRole('viewer', true)"
```

Reply:
Oh, I forgot to lint, thanks for the review! Fixed with a new commit